crates/librqbit/src/tracing_subscriber_config_utils.rs

```diff
@@ -112,6 +122,7 @@ pub fn init_logging(opts: InitLoggingOptions) -> anyhow::Result<InitLoggingResul
                 fmt::layer()
                     .with_ansi(false)
                     .with_writer(log_file)
+                    .and_then(if let Format::Json = opts.format { Some(fmt::layer().json())} else { None } )
```

Comment:
Not sure if it makes sense to change the log file to JSON at the same time as stdout.
One way to do this is 2 separate parameters:

--log-json (for stderr)
--log-file-log-json (for file log to be json)